grafana-plugin/.nvmrc

```diff
@@ -0,0 +1 @@
+20
```

Comment:
1. It's pity .nvmrc doesn't support comments but at least maybe we can use "v20" to indicate this s about node version?
2. It would be good to restrict this version in `package.json` using https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines ? wdyt?
3. Also it this case we stick to Node v20 on local but on on Github Actions and Drone it seems we use 18.16.0. I think we should be consistent here and use the same version. If not in this PR, can we create a task and add it to our [frontend tech backlog](https://github.com/grafana/oncall/issues/3733) pls?

Reply:
+1 for keeping the node versions the same on CI as local. A few spots that would need to be changed (this _should_ be all, but you'll want to double check):
- https://github.com/grafana/oncall/blob/dev/grafana-plugin/Dockerfile.dev#L1
- https://github.com/grafana/oncall-private/blob/dev/.drone.yml
- https://github.com/grafana/oncall/blob/dev/.drone.yml
- https://github.com/grafana/oncall/blob/dev/.github/workflows/linting-and-tests.yml
- https://github.com/grafana/oncall/blob/dev/.github/workflows/e2e-tests.yml